src/main/java/com/commercetools/sync/inventory/InventorySyncOptionsBuilder.java

```diff
@@ -0,0 +1,118 @@
+package com.commercetools.sync.inventory;
+
+import com.commercetools.sync.commons.BaseSyncOptionsBuilder;
+import com.commercetools.sync.commons.helpers.CtpClient;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Builder for creation of {@link InventorySyncOptions}
+ */
+public final class InventorySyncOptionsBuilder extends
+        BaseSyncOptionsBuilder<InventorySyncOptionsBuilder, InventorySyncOptions> {
+
+    private boolean ensureChannels = false;
+    private int parallelProcessing = 1;
+    private int batchSize = 30;
+
+    private InventorySyncOptionsBuilder(@Nonnull final CtpClient ctpClient) {
+        this.ctpClient = ctpClient;
+    }
+
+    /**
+     * Creates a new instance of {@link InventorySyncOptionsBuilder} given a {@link CtpClient}, as a param, that contains
+     * all the configuration of the CTP client.
+     *
+     * @param ctpClient wrapper that contains instance of the {@link io.sphere.sdk.client.SphereClientConfig} and
+     *                  {@link io.sphere.sdk.client.BlockingSphereClient}
+     * @return new instance of {@link InventorySyncOptionsBuilder}
+     */
+    public static InventorySyncOptionsBuilder of(@Nonnull final CtpClient ctpClient) {
+        return new InventorySyncOptionsBuilder(ctpClient);
+    }
+
+    /**
+     * Set option that indicates batch size for sync process. During the sync there is a need for fetching existing
+     * inventory entries, so that they can be compared with entries provided in input. That's why input is sliced into
+     * batches and then processed. It allows to reduce API calls by fetching existing inventories responding to
+     * inventories from processed batch in one call.
+     * E.g. value of 30 means that 30 entries from input list would be accumulated and one API call will be performed
+     * for fetching entries responding to them. Then comparision and sync are performed.
+     *
+     * This property is {@code 30} by default.
+     *
+     * @param batchSize int that indicates capacity of batch of processed inventory entries. Have to be positive
+     *                  or else will be ignored.
+     * @return {@code this} instance of {@link InventorySyncOptionsBuilder}
+     */
+    public InventorySyncOptionsBuilder setBatchSize(int batchSize) {
```

Comment:
@heshamMassoud I'm confused, you requested to use `final` where? On method's parameters, right?
@andrii-kovalenko-ct 

> it's a builder, it shouldn't be final

Here you mentioned that builder's **class** shouldn't be final, right? Why is that? `InventorySyncOptionsBuilder` isn't designed for inheritance (in opposite to `BaseSyncOptionsBuilder`), thus it's better to keep it `final` IMO.